src/types/webxr.d.ts

```diff
@@ -42,14 +42,104 @@ interface XR extends EventTarget {
   requestDevice(): Promise<XRDevice>;
 }
 
-interface XRSession {
+interface XRRigidTransform {
```

Comment:
This will be great once the spec is more stable and could find a home in some master typescript def list!